File: JS/E_Max.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split(/\s+/);

const size = Number(input[0]);
const arr = input.slice(1, size + 1).map(Number);

let mx = -Infinity;
for (const x of arr) mx = Math.max(x, mx);
console.log(mx);

File: C/E_Max.c
#include <stdio.h>

int main() 
{
    int size;
    scanf("%d", &size);
    int arr[size];
    for (int i = 0; i < size; i++)
        scanf("%d", &arr[i]);

    int mx = arr[0];
    for (int i = 1; i < size; i++)
        if (arr[i] > mx) mx = arr[i];
    
    printf("%d\n", mx);

    return 0;
}